sr_ur_robot_hw/src/sr_ur_robot_state_client.cpp

```diff
@@ -169,8 +195,9 @@ static void robot_state_client_connected_cb(uv_connect_t* connection_request, in
   ROS_ASSERT(0 == status);
   ROS_ASSERT(connection_request == &rs_client->connection_request_);
 
-  rs_client->buffer_.base = reinterpret_cast<char*>(malloc(2*sizeof(ur_robot_state)));
-  rs_client->buffer_.len  = 2*sizeof(ur_robot_state);
+  // Use the longest structure to allocate memory (ur_robot_state_v3_5)
+  rs_client->buffer_.base = reinterpret_cast<char*>(malloc(2*sizeof(ur_robot_state_v3_5)));
+  rs_client->buffer_.len  = 2*sizeof(ur_robot_state_v3_5);
```

Comment:
@toliver same as above?

Reply:
Not really (it is explained in the comment actually).

// Use the longest structure to allocate memory (ur_robot_state_v3_5)